demos/colab/colab_demo.ipynb

```diff
@@ -2,51 +2,50 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {
-    "colab_type": "text",
-    "id": "view-in-github"
-   },
+   "metadata": {},
    "source": [
-    "<a href=\"https://colab.research.google.com/github/tryolabs/norfair/blob/master/demos/yolov7/yolov7_demo.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+    "<a href=\"https://colab.research.google.com/github/tryolabs/norfair/blob/chore/colab-demo/demos/colab/colab_demo.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "id": "OMsRlH8QwFQd"
-   },
+   "metadata": {},
    "source": [
-    "# Simple object tracker\n",
+    "# Draw objects paths and track camera movement with Norfair\n",
+    "\n",
+    "Run a demo similar to the [Hugging Face Spaces Norfair demo](https://huggingface.co/spaces/tryolabs/norfair-demo) in this notebook.\n",
+    "\n",
+    "This demo uses the YOLOv7 model and shows the Norfair features to draw the object's paths and track camera movement.\n",
     "\n",
-    "Run a very simple car tracker on any Youtube video. This notebook is designed to run Norfair's [YOLOv7 demo](https://github.com/tryolabs/norfair/tree/master/demos/yolov7) on Google Colabratory.\n",
+    "Tack camera movement is useful to improve the tracker and also to keep the paths fixed although the camera movements.\n",
     "\n",
     "The demo will use the following video by default, but you can change which video you use by changing the youtube link in [this cell](#Download-Video-and-Preprocessing). We trim the video to only a few seconds due to limitations with video playback in Google Colab, but you can play with these limitations and see what you get.\n",
```

Comment:
youtube is mentioned but the video is downloaded from drive

Reply:
Great, old staff, removed the Youtube mention.